build.sbt

```diff
@@ -8,11 +8,11 @@ lazy val bintraySettings: Seq[Setting[_]] = Seq(
 )
 
 lazy val sharedSettings: Seq[Setting[_]] = Seq(
-  version := "0.1.0-SNAPSHOT",
-  scalaVersion := "2.11.6",
+  version := "0.1.1",
+  scalaVersion := "2.11.8",
   organization := "org.duhemm",
   resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-  libraryDependencies += "org.scalameta" %% "scalameta" % "0.1.0-SNAPSHOT",
+  libraryDependencies += "org.scalameta" % "scalameta_2.11" % "1.6.0",
```

Comment:
Why not just update scalameta's version number? It should still work with sbt's `%%`